DRAFT_homogeneous_impact_areas.Rmd

```diff
@@ -0,0 +1,426 @@
+---
+title: "-"
+output:
+  html_document
+editor_options: 
+  markdown: 
+    wrap: sentence
+---
+
+# Homogeneous Impact Areas
+
+<br />
+
+*Author and date:* 
+
+Anders Kolstad
+
+```{r, echo=FALSE}
+Sys.Date()
+```
+
+<br />
+
+<!-- Load all you dependencies here -->
+
+```{r setup, include=FALSE}
+library(knitr)
+knitr::opts_chunk$set(echo = TRUE)
+library(sf)
+library(tmap)
+library(dplyr)
+library(stars)
+library(ggplot2)
+library(units)
+library(ggpubr)
+library(DT)
+#library(osmplotr)
```

Comment:
Do we need these hashed out library calls? Does it add to the understanding or running of the code?

Reply:
Deleted it, and added :: when I use it